### ISSUE TITLE
OSError: Unable to open file

### ISSUE THREAD

**Issue by ahbon123** [idealo/image-quality-assessment#34]
I'm tring to evaluate around 3600 test images:
```
./predict  --docker-image nima-cpu --base-model-name MobileNet --weights-file $(pwd)/models/MobileNet/weights_mobilenet_technical_0.11.hdf5 --image-source $(pwd)/src/tests/test_images
```
and the error:

```
Using TensorFlow backend.
WARNING:tensorflow:From /usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
2019-06-13 09:24:55.396005: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-06-13 09:24:55.415707: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2208000000 Hz
2019-06-13 09:24:55.417119: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x4daabd0 executing computations on platform Host. Devices:
2019-06-13 09:24:55.417176: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/src/evaluater/predict.py", line 73, in <module>
    main(**args.__dict__)
  File "/src/evaluater/predict.py", line 44, in main
    nima.nima_model.load_weights(weights_file)
  File "/usr/local/lib/python3.5/dist-packages/keras/engine/topology.py", line 2658, in load_weights
    with h5py.File(filepath, mode='r') as f:
  File "/usr/local/lib/python3.5/dist-packages/h5py/_hl/files.py", line 394, in __init__
    swmr=swmr)
  File "/usr/local/lib/python3.5/dist-packages/h5py/_hl/files.py", line 170, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 85, in h5py.h5f.open
OSError: Unable to open file (file read failed: time = Thu Jun 13 09:24:57 2019
, filename = '/src/weights.hdf5', file descriptor = 3, errno = 21, error message = 'Is a directory', buf = 0x7ffff603d840, total read size = 8, bytes this sub-read = 8, bytes actually read = 18446744073709551615, offset = 0)
```
Thanks.

**Comment by clennan** [comment]
I am not sure what the error is here, are all the paths to the model and image dir set correctly?

**Comment by razbrim** [comment]
Hi,
I get the same error.
were you able to solve it?

**Comment by suissemaxx** [comment]
I could solve the error by **removing all spaces from my directory path**.

I had this path:

`/Users/peterpeter/Documents/_ Code/_nima/image-quality-assessment-master`

After removing the space in `_ Code` and changing it to `_Code` I successfully could run the `predict` command.

**Comment by fliodhais** [comment]
Hi, may I know where is the location of this path? 

I assumed you meant where the repo was living on my machine but I followed the original repo's instruction (loading with docker) and I can't access that path? 

If you mean the path in the code of the repo, may I know how you adjusted it? 
![image](https://user-images.githubusercontent.com/40507267/142757238-2cc9abe2-d9a7-40cd-8c5b-a4444a540e6a.png)
I found this line in the predict.py file but not sure how that affects anything.

Interestingly enough, when i went to search at /src/weights.hdf5, I don't see anything.
